### PR TITLE
feat(Cosmetic): Adapt UI to BioChatter styles

### PR DESCRIPTION
**Pull request by agdiaz**
This PR addresses the goals described in issue #1

- [X] Edit page title
- [X] Add BioChatter favicon
- [X] Update titles, subtitles, and other texts
- [X] Add logo in sidebar's header
- [X] Replace the Mask concept with Persona
- [X] Update Github link to connect app to this repository
- [X] Add link to official docs 
- [X] Add About page and button

image

image

image

### CODE REVIEW

File: chatgse/app/locales/cn.ts [hunk 1]
```diff
@@ -342,7 +342,7 @@ const cn = {
   },
   Store: {
     DefaultTopic: "新的聊天",
-    BotHello: "有什么可以帮你的吗",
+    BotHello: "Hello! I am the model's assistant. For more explanation, please see the About text in the sidebar. We will now be going through some initial setup steps together. To get started, could you please tell me your name?",
```
Comment (slobentanzer) [inline chatgse/app/locales/cn.ts:345]: I would revert to the defaults and include this general introduction in a "home page" of sorts, which is shown in the beginning and can be accessed via a help button afterwards. The home page could include the info section, about section, and the note about the paper in the original ChatGSE.

File: chatgse/app/locales/cn.ts [hunk 2]
```diff
@@ -373,6 +373,22 @@ const cn = {
   FineTuned: {
     Sysmessage: "你是一个助手",
   },
+  About: {
```
Comment (slobentanzer) [inline chatgse/app/locales/cn.ts:376]: These we can keep for now, although chinese would probably be better at some point

File: chatgse/public/prompts.json [hunk 3]
```diff
@@ -1,124 +1,29 @@
 {
```
Comment (slobentanzer) [inline chatgse/public/prompts.json:1]: Maybe revert this as well; I think the prompts were kind of interesting. Maybe we can add some biomed prompts at the very top, and put this general introduction also on the new "home page". The home page can be in another PR if it does not take priority now.

File: chatgse/app/locales/cn.ts (hunk 1 above)
Comment (agdiaz) [reply chatgse/app/locales/cn.ts]: Reverted here and in the English locale file

File: chatgse/app/locales/cn.ts (hunk 2 above)
Comment (agdiaz) [reply chatgse/app/locales/cn.ts:376]: I translated them from English to simplified Chinese using Google Translate as a first version of these messages

File: chatgse/public/prompts.json (hunk 3 above)
Comment (agdiaz) [reply chatgse/public/prompts.json:1]: Reverted!
